dockerhub_doc_config_update.sh

```diff
@@ -35,7 +35,7 @@ fi
 wget -q -O official-eclipse-temurin https://raw.githubusercontent.com/docker-library/official-images/master/library/eclipse-temurin
 
 # TODO When we have Apline musl builds we should add alpine to the follow list
```

Comment:
:eyes:

Reply:
We're working on backporting to Java 11 and 8 - it'll be several weeks but we'll get there!

Reply:
Oh, I see, comment is redundant at least with regards to this config.